astropy/utils/report_info.py

```diff
@@ -0,0 +1,98 @@
+r"""
+A helper script to automate system info in bug reports.
+
+This can be run in two fashions either by downloading the latest version of this
+script and running it on the fly
+```
+curl --silent https://raw.githubusercontent.com/astropy/astropy/main/astropy/utils/report_info.py | python
+```
```

Comment:
I'd remove this entirely. It is not possible to provide a simple shell one-liner that will work on all the supported platforms.